.github/actions/program/token-auth-rules/action.yml

```diff
@@ -0,0 +1,13 @@
+
+name: Token Authorization Rules and Rooster Programs
+description: Gets mpl-token-auth-rules and rooster
+runs:
+  using: 'composite'
+  steps:
+    # Get Token Auth Rules program
+    - name: Get Token Auth Rules and rooster program
+      run: |
+        mkdir -p test-programs
+        solana program dump auth9SigNpDKz4sJJ1DfCTuZrZNSAgh9sFD3rboVmgg test-programs/mpl_token_auth_rules.so
```

Comment:
Nice!

Reply:
Can you specify the network as well, just so it's clear?